network.go

```diff
@@ -2021,3 +2053,80 @@ func (c *controller) getConfigNetwork(name string) (*network, error) {
 
 	return n.(*network), nil
 }
+
+func (n *network) createLoadBalancerSandbox() error {
+	sandboxName := n.name + "-sbox"
+	sbOptions := []SandboxOption{}
+	if n.ingress {
+		sbOptions = append(sbOptions, OptionIngress())
+	}
+	sb, err := n.ctrlr.NewSandbox(sandboxName, sbOptions...)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err != nil {
+			if e := n.ctrlr.SandboxDestroy(sandboxName); e != nil {
+				logrus.Warnf("could not delete sandbox %s on failure on failure (%v): %v", sandboxName, err, e)
+			}
+		}
+	}()
+
+	endpointName := n.name + "-endpoint"
+	epOptions := []EndpointOption{
+		CreateOptionIpam(n.loadBalancerIP, nil, nil, nil),
+		CreateOptionLoadBalancer(),
+	}
+	ep, err := n.createEndpoint(endpointName, epOptions...)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err != nil {
+			if e := ep.Delete(true); e != nil {
+				logrus.Warnf("could not delete endpoint %s on failure on failure (%v): %v", endpointName, err, e)
+			}
+		}
+	}()
+
+	if err := ep.Join(sb, nil); err != nil {
+		return err
+	}
+	return sb.EnableService()
```

Comment:
As you know EnableService will handle the Service Discovery portion of the Sandbox and distributes the SD information to other nodes in the cluster. What is the purpose of distributing the `endpointName` to loadbalancerIP mapping throughout the cluster ?

Reply:
I see. I misunderstood what that function did.  
Do we need this for the ingress network?  
The previous code called EnableService() when setting up the ingress network
(https://github.com/moby/moby/blob/9be245f438f9fb2eaeb7891673b16aed9262a192/daemon/network.go#L204)

Reply:
ping @mavenugo .  Should we remove the call to EnablesService?  I don't see any need for it.  Although we could keep it for ingress because it was like that before.

Reply:
Its okay. I think you are correct that ingress had this call before and it is better to leave it in there.